geode-core/src/test/java/org/apache/geode/distributed/internal/DistributionConfigJUnitTest.java

```diff
@@ -140,7 +140,7 @@ public void testGetAttributeNames() {
     // are.
     assertEquals(36, boolList.size());
     assertEquals(35, intList.size());
-    assertEquals(88, stringList.size());
+    assertEquals(89, stringList.size());
```

Comment:
I generally update every test I touch to use AssertJ instead of JUnit Assertions. There's a great IntelliJ plugin that will do the bulk of the work for you called **Assertions2AssertJ**. For example, using this plugin adds a new menu item to "**Refactor -> Convert Assertions to AssertJ**".

Reply:
Thanks, I didnt know that plugin!